src/Error/ExceptionRenderer.php

```diff
@@ -158,13 +158,26 @@ public function render()
             $this->controller->response->header($exception->responseHeader());
         }
         $this->controller->response->statusCode($code);
-        $this->controller->set([
+        $viewVars = [
             'message' => $message,
             'url' => h($url),
             'error' => $exception,
             'code' => $code,
             '_serialize' => ['message', 'url', 'code']
-        ]);
+        ];
+        if ($isDebug) {
+            $rawTrace = $exception->getTrace();
+            $trace = [];
+            foreach ($rawTrace as $line) {
+                if (isset($line['args'])) {
+                    unset($line['args']);
+                }
+                $trace[] = $line;
+            }
+            $viewVars['trace'] = $trace;
```

Comment:
@ADmad I preferred the array because this would allow you to customize it on the view side a little better than a string.